### DISCUSSION TITLE
GitHub Support does not respond - Account suspended

### DISCUSSION THREAD

**Discussion by h3li0p4us3-moharami** [community/community#143432] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

Hey everyone,

I'm pretty baffled right now. My GitHub account, h3li0p4us3.  was suddenly suspended 5 days ago. I haven't received any notification or explanation. It's really frustrating, especially since I rely on GitHub for work and personal projects.

Any advice on how to get this resolved quickly? I've already submitted a support ticket but haven't heard back yet.

my suspended account username: github.com/h3li0p4us3
my first ticket's ID: 3075070

**Comment by davevad93** [comment, marked as answer]
Hi @h3li0p4us3-moharami ,

Since yours is an account related issue, the only way to try to resolve it by contacting support, as you have done, but be aware that they can take a long time to respond, even weeks, because they have to deal with lots of daily requests, so be patient. I hope you can get a reply and solution as soon as possible.

**Reply by h3li0p4us3-moharami** [reply]
Thank you, @davevad93!
I hope they respond quickly, as I’ve been sending my resume to a lot of job positions. It's pretty important! 😄

**Reply by ibrahimm7004** [reply]
Were you able to get a reply? The same thing happened to me, and I have not received any kind of reply at all..it's been around 2 months now
